src/constants/index.js

```diff
@@ -0,0 +1,10 @@
+export const HOME_PAGE_TITLE = 'Dreams Will Come True';
+
+export const HOME_INFORMATION_TEXT =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minimveniam consequat.';
+
+export const TRANSATION_MAIN_TEXT =
+  'With the Right Organization, You Can Give A Hand True Needer';
+
+export const TRANSATION_SUB_TEXT =
+  "I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to add your own content and make changes to the font. Feel free to drag and drop me anywhere you like on your page. I’m a great place for you to tell a story and let your users know a little more about you.";
```

Comment:
Same goes here, should we take them inside a folder related to homepage constants or does the team want to use this way too ?

Reply:
I updated them 👍🏼  We didn't discuss about it but we can talk today 👍🏼